CMakeLists.txt

```diff
@@ -164,6 +169,10 @@ else()
     MESSAGE(FATAL_ERROR "Unknown CRYPTO")
 endif()
 
+if ((X509_IGNORE_CRITICAL STREQUAL "ON") AND (CRYPTO STREQUAL "openssl"))
```

Comment:
Also add this option to MbedTLS.

Reply:
Thanks. I have added the flag for Mbedtls.
https://github.com/DMTF/libspdm/blob/93459f4720feab6e08540aeefe29a7c761272db2/CMakeLists.txt#L172-L178

The flag is from:
https://github.com/DMTF/libspdm/blob/81791b8b958085cd9ed2f1a51336de20d0eb81bd/os_stub/mbedtlslib/include/mbedtls/config.h#L2203-L2213